viewer/viewer-awt/src/main/java/org/icepdf/ri/common/SwingViewBuilder.java

```diff
@@ -990,6 +991,15 @@ public JMenuItem buildSearchMenuItem() {
         return mi;
     }
 
+    public JMenuItem buildAdvancedSearchMenuItem() {
+        final JMenuItem mi = makeMenuItem(messageBundle.getString("viewer.toolbar.search.advanced.label"), buildKeyStroke(KeyEventConstants.KEY_CODE_SEARCH, KeyEventConstants.MODIFIER_ADVANCED_SEARCH));
+        if (viewerController!=null && mi!=null){
+            viewerController.setAdvancedSearchMenuItem(mi);
+        }
+        mi.setPreferredSize(new Dimension(0,0));
```

Comment:
I'm in favour of just adding the menu item and not hiding it.   There no obvious way to get to the advance search bar via the toolbar other then right click on the filter on the search panel.  Also has the benefit of showing the keyboard shortcuts to the end user.